docs/browser-acceptance-tests.md

```diff
@@ -118,23 +118,36 @@ Scenario: User logs in
 
 Sauce Labs can be used to run tests remotely in the cloud.
 
-1. Log into [https://saucelabs.com/account](https://saucelabs.com/account).
-
-2. Update and uncomment the `SAUCE_USERNAME`, `SAUCE_ACCESS_KEY`,
-   and `SAUCE_SELENIUM_URL` values in your `.env` file.
-   The access key can be found on the Sauce Labs
-   [user settings page](https://saucelabs.com/beta/user-settings).
-
-3. Reload the settings with `source .env`.
-
-4. Run the tests with `gulp test:acceptance --sauce`.
-
-5. Monitor progress of the tests
+### For new Sauce Labs users
+
+1. [Follow the instructions here](https://github.com/cfpb/development/blob/master/guides/browser-testing-with-sauce-labs.md) in the "Sign up for saucelabs.com" and "Set up Sauce Connect to test URLs on local and dev servers" sections. The remaining sections are not required to run cfgov-refresh browser tests, but are recommended if you want to use Sauce Labs to manually QA in supported browsers.
+1. Edit your cfgov-refresh `.env` file. Uncomment the `export` statements in the "Sauce Labs section of the file, and update them as follows:
+    -  `SAUCE_USERNAME`: Your saucelabs.com username
```

Comment:
TRIVIAL: Extra space before dash

Reply:
I committed a change to remove extra space before dash for SAUCE_USERNAME in browser-acceptance-tests.md